mautic/mautic
# New campaign action: Update company

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Automated tests included? |
| Related user documentation PR URL | https://github.com/mautic/documentation/pull/269
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/5606
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

This bugfix/feature has been sponsored by @Webmecanik

[//]: # ( Required: )
#### Description:
New campaign action will update contacts primary company fields based on community request https://github.com/mautic/mautic/issues/5606

![image](https://user-images.githubusercontent.com/462477/36399778-bfd2e790-15cd-11e8-9614-d03ab842fd6a.png)


#### Steps to test this PR:
1.  Create contact and add company to contact
2. Create campaign, add contact to campaign and add new Update contacts primary company
3. Run campaign and see if primary company was updated
